src/js/controllers/ChooseView.ts

```diff
@@ -0,0 +1,28 @@
+import { NumberCalculate } from "./NumberCalculate";
+import { RangeCalculate } from "./RangeCalculate";
+
+export class ChooseView {
+  public radioPanel: HTMLElement;
+  public container: HTMLElement;
+
+  constructor() {
+    this.radioPanel = document.getElementById("radio-panel");
+    this.container = document.getElementById("container");
+
+    this.radioPanel.addEventListener("click", (event: any) => {
+      const target = <HTMLElement>event.target;
+      if (!this.handler[target.id]) return;
+      this.container.innerHTML = "";
+      this.handler[target.id]();
+    });
+  }
+
+  public handler: { [key: string]: Function } = {
```

Comment:
You can use Record to as a typical object description 
And I suggest to use `() => void` as more strict than general Function type
```suggestion
  public handler: Record<string, () => void> = {
```